Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py

```diff
@@ -94,7 +94,7 @@ def DictStartVisit(self, obj):
         output_dir = os.environ["DICT_DIR"] + "/channels"
         if not (os.path.isdir(output_dir)):
             os.makedirs(output_dir)
-        pyfile = output_dir + "/" + obj.get_name() + ".py"
+        _ = output_dir + "/" + obj.get_name() + ".py"
```

Comment:
same here

Reply:
I made a lot of switches from variable -> _ but left the line in.  Given the overall quality of the code base, I cannot be sure that there aren't side-effects to function calls and what not.

Thus clean ups more substantial will require a full refactor.

Reply:
Yeah, there may be some unexpected affects, but if that's the case can't you just directly call the functions you're concerned have side-effects?

Also, just scrolling through the changes, most of them seem this one, where removing the entire line seems pretty safe (unless get_name is doing something insane).

I'd argue this cleanup is pointless without actually removing the offending lines (other than appeasing the linter), but if it doesn't make sense to invest the effort in removing these lines now, I guess at least clearly marks the lines for cleanup later.